### ISSUE TITLE
Criar Lista de Requisitos para Matrícula

### ISSUE THREAD

**Issue by braganholo** [gems-uff/sapos#174]
Criar o conceito de Requisito, que pode ser cadastrado pela secretaria. 

Cada matrícula teria que cumprir esses requisitos. Portanto, os requisitos apareceriam associados à matricula, com um checkbox, ou combo-box com SIM ou NÃO para que a secretaria marque quem já cumpriu um determinado requisito.

**Comment by braganholo** [comment]
Exemplos de requisitos: entrega de documentação de matrícula; entrega da versão final da dissertação; publicação de artigo A1, A2, B1.

**Comment by danielpcampagna** [comment]
Só para ver se eu entendi: Seria adicionar um combo-box, contendo todos os possíveis Requisitos, e outro, contendo a informação se já foi cumprido, na tela de inserir/editar matrícula?

**Comment by leomurta** [comment]
Daniel, esse aqui não é bug, mas sim feature. Se tiver bug em aberto, dê prioridade. Para esse aqui acho melhor marcar um dia para uma conversa, para vc fazer o levantamento de requisitos.

**Comment by leomurta** [comment]
A lista de bugs está aqui: https://github.com/gems-uff/sapos/milestones/4.2.x

**Comment by braganholo** [comment]
Sim, mas depois que você adiciona um requisito como cumprido, ele fica aparecendo lá (como uma inscrição em turma ou uma realização de etapa -- veja como isso é feito na tela de matrículas). 

Além disso, teria que ter uma tela onde eu pudesse cadastrar esses requisitos.

**Comment by braganholo** [comment]
Associar também um campo de observação vinculado do check-box na tela de matrícula. Isso serviria, por exemplo, para a secretaria cadastrar o título do artigo, se o requisito for um artigo. Adicionar também uma data que indicaria a data de conclusão do requisito.

**Comment by leomurta** [comment]
Acho que isso não é mais necessário depois que a inscrição online foi feita, né? 
@braganholo , se vc concordar, fecha essa issue por favor.

**Comment by braganholo** [comment]
Não tem relação com a inscrição online. Na verdade a proposta aqui era uma simplificação do conceito de etapas, para separar um pouco as coisas. Por exemplo, Pedido de Banca continuaria sendo uma etapa (pode ser prorrogada), mas submissão de artigo seria um requisito que o aluno tem que cumprir antes de defender. No entanto, entendo que isso não é essencial e que as etapas atendem o que precisamos. Portanto, por mim podemos fechar essa issue se vcs concordarem.

**Comment by leomurta** [comment]
Ah, entendi. A ideia seria separar etapa de requisito e passar a ter o conceito de requisito explícito no sapos. Acho legal termos isso sim, mas demandaria refinarmos como seria, até pq requisitos mudam com o tempo, então seria necessário podermos definir a validade do requisito (alunos que ingressaram do ano X a Y). Vou deixar aberta, mas passei para milestone 7.x, que indica menor prioridade por ser algo mais complexo.
